firebase-crashlytics/firebase-crashlytics.gradle

```diff
@@ -99,6 +99,8 @@ dependencies {
     testImplementation(libs.robolectric)
     testImplementation(libs.truth)
     testImplementation(project(":integ-testing"))
+    testImplementation("io.mockk:mockk:1.13.8")
```

Comment:
this will be removed, was using to attempt to mock a few things but I found it was not needed